sources/engine/Stride.Engine/Engine/Processors/InstancingProcessor.cs

```diff
@@ -87,18 +87,19 @@ private void UpdateInstancing(InstancingComponent instancingComponent, Instancin
             if (instancingComponent.Enabled && instancingComponent.Type != null)
             {
                 var instancing = instancingComponent.Type;
+                var modelComponent = instancingData.ModelComponent;
 
                 // Calculate inverse world and bounding box
                 instancing.Update();
 
-                if (instancingData.ModelComponent != null && instancing.InstanceCount > 0)
+                if (modelComponent != null && modelComponent.Model != null && instancing.InstanceCount > 0)
```

Comment:
`modelComponent?.Model != null`?

Reply:
yes, that is indeed much shorter in terms of changes. But some lines got a little shorter with the local variable. just a matter of taste. if you prefer the `?` operator I'll change it.

Reply:
There's no local variable being assigned in that comparison
xen2 was saying that `modelComponent != null && modelComponent.Model != null` can be written as `modelComponent?.Model != null`
No behaviour change there, no change to what variables get assigned

Reply:
Oh, i see it now... Thanks! :-)